workers/config/src/main/kotlin/ConfigComponent.kt

```diff
@@ -50,7 +50,7 @@ class ConfigComponent : EndpointComponent<ConfigRequest>(ConfigEndpoint) {
 
         val runId = message.payload.ortRunId
         val responsePayload = when (val result = runBlocking { configWorker.run(runId) }) {
-            is RunResult.Success -> {
+            is RunResult.Success, RunResult.Ignored -> {
```

Comment:
How about just returning `RunResult.Success` also in case the config script is missing? This should be a normal case, so does not require any special return value.

Reply:
Yes, that was the other option I considered, I wasn't sure if there was a reason to us `Ingored` that I didn't see.

Reply:
Not sure why `Ignored` was used initially, but I think `Success` would be the correct result. Other workers use `Ignored` when they cannot find the current ORT run. This is not the case here.